.ddev/commands/web/typo3

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+## Description: Execute TYPO3 Command (CLI)
+## Usage: typo3
+## Example: "ddev typo3 list" or "ddev typo3 help" or "ddev typo3 --help"
+
+.Build/bin/typo3 $@
```

Comment:
for this command to work (without line ending issues) .gitattributes from the core would need to be updated as well, right?

Anyway, if we see this ddev config as help for TYPO3 contributors, can you elaborate how this addition is helpful? I never needed to execute this binary :)

Reply:
I add this by default to all my TYPO3 projects. It's not needed here and can be removed if you like...

Reply:
This command is removed, but I changed the .gitattributes for the typo3cms command